tests/ensure_provider_tests.py

```diff
@@ -88,6 +88,12 @@
         EXTRA_RUN_CONDITION: "ansible_distribution_major_version == '7'",
     },
     "playbooks/tests_wireless_plugin_installation.yml": {},
+    "playbooks/tests_wireless_wpa3_owe.yml": {
+        "comment": "# OWE has not been exposed via D-Bus on RHEL 7(dist-tag) and \
```

Comment:
What do you mean with D-Bus here?

Reply:
I just want to indicate that OWE has not been supoorted by `NetworkManager-1.18.8` on RHEL 7.

Reply:
Then please specify it is not supported by `NetworkManager-1.18.8` on RHEL7, I think that is more clear that `has not been exposed via D-Bus on RHEL7`

Reply:
Comment fixed.